definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1426,5 +1426,12 @@
     "CaseEventID": "FR_updateOrganisation",
     "UserRole": "caseworker-divorce-systemupdate",
     "CRUD": "R"
+  },
```

Comment:
Do solicitors and judiciary not require read access?

Reply:
its only for caseworker

Reply:
addd read only accesss for solicitors and judiciary

Reply:
cheers